assets/scss/_mixins.scss

```diff
@@ -20,6 +20,36 @@
   @return if(breakpoint-min($name, $breakpoints) == null, '', '-#{$name}');
 }
 
+@mixin bulletpoints() {
+  ul {
+    &.dashed {
+      li {
+        &::before {
+          margin-left: 0;
+          content: '';
+        }
+      }
+    }
+
+    li {
+      position: relative;
+
+      &::before {
+        content: '';
+        position: absolute;
+        display: inline-flex;
+        height: 2px;
+        width: 20px;
+        top: 11px;
+        left: -20px;
```

Comment:
Die 20px hängen doch zusammen oder? Kann man vielleicht in einen Wert auslagern.